### ISSUE TITLE
Problematic default colors

### ISSUE THREAD

**Issue by jzaefferer** [tldr-pages/tldr-cpp-client#1]
I get this output, here for `tldr wc`, on iTerm2/OSX:

![...](http://bassistance.de/i/d55534.png)

Maybe my console's color scheme is at fault, but since there doesn't seem to be a way to change the colors `tldr` is using (unlike the [node client](https://github.com/tldr-pages/tldr-node-client#configuration), but that one [doesn't work at all](https://github.com/tldr-pages/tldr-node-client/issues/8)) and there is no documentation about what color set is uses, I don't know how to fix this on my end.

**Comment by nemecec** [comment]
+1

As a workaround, I just switched to dark background (light background has been a problem also with other tools).

**Comment by Leandros** [comment]
Should've been resolved due to the help of @rastersize!

**Comment by VijayaKrishnaKasula** [comment]
TLDR Uses the term colors from https://pypi.python.org/pypi/termcolor
You can edit the `DEFAULT_COLORS` in `whch tldr.py`

I simply removed the backgrounds by deleting all words that match `perl_re(' on_\w+')`
and changed the white color to gray

It worked for me greatly on solarized_light or white background

Output on solarized_light
![image](https://user-images.githubusercontent.com/7437418/33237208-00364cce-d293-11e7-9bc0-97067d520050.png)

**Comment by Leandros** [comment]
This is not the python client.
